crates/core_simd/src/vector.rs

```diff
@@ -99,7 +99,7 @@ use crate::simd::{
 // directly constructing an instance of the type (i.e. `let vector = Simd(array)`) should be
 // avoided, as it will likely become illegal on `#[repr(simd)]` structs in the future. It also
 // causes rustc to emit illegal LLVM IR in some cases.
-#[repr(simd)]
+#[repr(simd, packed)]
```

Comment:
What is the plan for `simd` without `packed`? Miri currently ICEs there. If portable-simd doesn't need it then do we need to have it at all? Can we just make `simd` itself have the behavior that `simd, packed` now has?

Reply:
`simd` without `packed` is used by stdarch. portable-simd might use it in the future too, though imo probably won't.

Reply:
AFAIK stdarch only uses power-of-2 vectors, where `packed` makes no difference?